core/tests/karma.conf.ts

```diff
@@ -56,7 +56,8 @@ module.exports = function(config) {
       'local_compiled_js/core/templates/dev/head/**/*-e2e.js',
       'local_compiled_js/extensions/**/protractor.js',
       'backend_prod_files/extensions/**',
-      // add these back after the templateCache is repaired
+      // TODO(vojtechjelinek): add these back after the templateCache
+      // is repaired
```

Comment:
Can you elaborate further (in comments) what you mean by "templateCache is repaired"?

Reply:
It is connected to the `/extensions` and the fact that the webpack is not yet introduced there yet, and after we introduce the webpack into `/extensions` this will be fixed too.

Reply:
OK -- can you explain this more clearly in the file itself? For the benefit of future devs. Maybe point to a GitHub issue number too.

Reply:
Done.